src/extensions/default/BrambleUrlCodeHints/camera/interface.js

```diff
@@ -85,10 +89,14 @@ define(function (require, exports, module) {
 
         function persistPhoto() {
             var data = self.photo.data;
+            var data2 = self.photo.data2;
             if(!data) {
+              alert("no data to save");
```

Comment:
For debugging, prefer `console.log()` to `alert()` so that you can still keep working.  You just need to open your dev tools.